docs/deployment-guide.md

```diff
@@ -275,6 +275,10 @@ By default the playbook will deploy and configure a Postgres server for you. Tha
 
 > This server also requires to NOT have a sudo configuration with `requirestty` set.
 
+Please be aware that a change in Postgres configuration (e.g. pg_hba) can
```

Comment:
Sorry I think my previous comment was unclear. Actually the reset of permissions  is just another thing that makes the whole playbook not usable outside of maintenance slots. Hence this comment applies are the playbook level not only for db tasks

Reply:
wait, I was thinking that you are referring to restart problems in general and not specific to this pr: here I am changing permissions on the fly and no service will be restarted.

in any case I think it's fine to add something to upper docs